gnomad_constraint/pipeline/constraint_pipeline.py

```diff
@@ -432,21 +446,28 @@ def main(args):
             # for XX/XY in the future).
             for r in regions:
                 logger.info(
-                    "Applying %s plateau and autosome coverage models and computing"
-                    " expected variant count and observed:expected ratio...",
+                    "Applying %s plateau and autosome coverage models (if specified)"
+                    " and computing expected variant count and observed:expected"
+                    " ratio...",
                     r,
                 )
                 oe_ht = apply_models(
                     getattr(res, f"preprocessed_{r}_exomes_ht").ht(),
                     getattr(res, f"preprocessed_{r}_context_ht").ht(),
                     mutation_ht,
                     getattr(res, f"model_{r}_plateau").he(),
-                    getattr(res, "model_autosome_par_coverage").he(),
+                    (
+                        getattr(res, "model_autosome_par_coverage").he()
+                        if not args.skip_coverage_model
+                        else None
+                    ),
                     max_af=max_af,
                     pops=pops,
                     obs_pos_count_partition_hint=args.apply_obs_pos_count_partition_hint,
                     expected_variant_partition_hint=args.apply_expected_variant_partition_hint,
                     custom_vep_annotation=custom_vep_annotation,
+                    cov_cutoff=args.lower_cov_cutoff,
+                    low_coverage_filter=low_coverage_filter,
```

Comment:
this is confusing to me since it looks like there are two different low coverage cutoffs